backend/src/main/scala/cromwell/backend/BackendLifecycleActor.scala

```diff
@@ -15,21 +15,14 @@ object BackendLifecycleActor {
    * Commands
    */
   trait BackendWorkflowLifecycleActorCommand extends BackendWorkflowLifecycleActorMessage
-  case object AbortWorkflow extends BackendWorkflowLifecycleActorCommand
-  final case class AbortJob(jobKey: BackendJobDescriptorKey) extends BackendWorkflowLifecycleActorCommand
+  case object AbortWorkflowCommand extends BackendWorkflowLifecycleActorCommand
+  case object AbortJobCommand extends BackendWorkflowLifecycleActorCommand
```

Comment:
Yeah, I think that "`abort()` within the `WorkflowExecutionActor`" is going to need to be a new ticket following @gauravs90's scoping-out of execution.